.npmrc

```diff
@@ -0,0 +1,3 @@
+save-prefix=''
+# Avoids a sub-dependency of Source failing to resolve and breaking the build
+shamefully-hoist=true
```

Comment:
[shamefully-hoist](https://pnpm.io/npmrc#shamefully-hoist)

aiui the docs suggest the hoisting issue is coming from our code or tooling rather than a dependency in node_modules?

Reply:
Could this be a straightforward-ish fix in source? @sndrs might be interested in this issue

Reply:
Not actually a Source issue in the end; I discovered in our Webpack config we had symlink resolution disabled, so this was probably just the first of many potential problems that the build hit trying to find various modules.

Reply:
sorry was on leave last week. yeah we did the same in dcr. we found whenever we had to do something to get pnpm to work, it was almost always because we were already doing the wrong (or a wierd) thing somewhere.